Source/Core/Core/HW/Memmap.cpp

```diff
@@ -189,6 +204,39 @@ static u32 GetFlags()
 
 void Init()
 {
+  auto GetMEM1Size = []()->u32 {
+    if (SConfig::GetInstance().m_RAMOverrideEnable == true)
+      // Configurable MEM1 Size
+      return SConfig::GetInstance().m_MEM1Size;
+    else
```

Comment:
This else can be removed.

Reply:
In my opinion, the else makes the flow of the function more clear at a glance, that is why I left it in.  If you still disagree, I can remove it.